get-docker-lang-versions.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+printf 'Docker image,PHP version,NodeJS version,Python version\n' >> docker-lang-versions.txt
+
+while IFS= read -r line; do
```

Comment:
However, might I ask for more readability by renaming line to something like image or container or whatever the appropriate docker term would be?

Reply:
In this case, "image" would be the appropriate term, and I'm fine with renaming `$line` to `$image` in this loop. 👍